DigitalOffice/Tests/UserServiceTests/CommandsTests/UserCreateCommandTests.cs

```diff
@@ -12,7 +11,7 @@
 using LT.DigitalOffice.UserService.RestRequests;
 using LT.DigitalOffice.UserService.Validators;
 
-namespace LT.DigitalOffice.UserServiceUnitTests.Commands
```

Comment:
We need to negotiate this agreement again today on skype.

Reply:
It is necessary to change the name from CommandsTests to Commands.

Reply:
Fixed